MovieQuiz/Services/StaticService.swift

```diff
@@ -0,0 +1,76 @@
+import Foundation
+
+final class StatisticServiceImplementation: StatisticService {
+    
+    private let userDefaults = UserDefaults.standard
+    
+    private var correct: Int {
+        get {
+            userDefaults.integer(forKey: Keys.correct.rawValue)
+        }
+        set {
+            userDefaults.set(newValue, forKey: Keys.correct.rawValue)
+        }
+    }
+    
+    private var total: Int {
+        get {
+            userDefaults.integer(forKey: Keys.total.rawValue)
+        }
+        set {
+            userDefaults.set(newValue, forKey: Keys.total.rawValue)
+        }
+    }
+    
+    
+    
+    
+    var totalAccuracy: Double {
+        get {
+            return (Double(correct) / Double(total)) * 100
+        }
```

Comment:
Когда у вас есть только гетер, и сеттер не нужен, можно опускать конструкцию get { }
```suggestion
            return (Double(correct) / Double(total)) * 100
```